.coveralls.yml

```diff
@@ -0,0 +1,2 @@
+repo_token: MCQk267Y43SUWrhGJsYyGQDujK4yHxdcZ
```

Comment:
The token should be kept secret AFAIK

Reply:
The document said so and I don't think that's a secret.

Reply:
Removed. Saved into travis environment variable.